enterprise/frontend/src/embedding-sdk/cli/steps/pick-database-tables.ts

```diff
@@ -36,51 +39,109 @@ export const pickDatabaseTables: CliStepMethod = async state => {
       { retries: 10, delay: 1000 },
     );
   } catch (error) {
-    const reason = error instanceof Error ? error.message : String(error);
-    const message = `Cannot fetch database schema. Reason: ${reason}`;
     spinner.fail();
 
-    return [{ type: "error", message }, state];
+    return [cliError("Cannot fetch database schema", error), state];
   }
 
-  const tables: Table[] = [];
+  const tablesWithoutMetadata: Table[] = [];
```

Comment:
The first call to `/api/database/<db-id>/schema/<schema-key>` gets a list of tables within that schema without field metadata.